test/specs/plugin.filler.tests.js

```diff
@@ -223,14 +223,14 @@ describe('Plugin.filler', function() {
 			});
 
 			expect(decodedFillValues(chart)).toEqual([
-				'start', // 'origin'
-				0,       // 1 - 1
-				1,       // 1
-				1,       // 3 - 2
-				5,       // 4 + 1
-				7,       // 5 + 2
-				5,       // 6 - 1
-				'end',   // 'end'
+				'start',	// 'origin'
```

Comment:
Crap, this option broke Code Climate because it's only available in eslint v4 and Code Climate only supports eslint v3 :-(  https://github.com/codeclimate/codeclimate-eslint/issues/283